tests/semgrep/scripts/python_portability.py

```
@@ -0,0 +1,242 @@
+"""Static-analysis fixtures only; never execute these example calls."""
+
+import io
+import os
+import subprocess
+import tempfile
+from collections.abc import Mapping, Sequence
```

Ruff (0.16.3)

[warning] Move standard library import `collections.abc.Mapping` into a type-checking block

Move into type-checking block

(TC003)